cluster/cluster.yaml

```diff
@@ -1173,9 +1104,9 @@ Resources:
                   ],
                   "Condition": {
                     "StringEquals": {
-                      "aws:ResourceTag/kubernetes.io/cluster/{{.Cluster.ID}}": "owned",
+                      "aws:ResourceTag/kubernetes.io/cluster/{{.Cluster.Name}}": "owned",
+                      "aws:RequestTag/kubernetes.io/cluster/{{.Cluster.Name}}": "owned",
```

Comment:
duplicate line?

Reply:
It's just because I moved line 1178 up here instead. It made more sense in the original PR where I used if guards. I can also revert this part so the lines don't jump.

Reply:
If it's just moving the lines then it's fine. I thought we changed something otherwise.